misc/io.github.efogdev.mpris-timer.metainfo.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.efogdev.mpris-timer</id>
+	<url type="homepage">https://github.com/EfogDev/mpris-timer</url>
+	
+	<name>MPRIS Timer</name>
+	<summary>A timer app utilizing libadwaita and MPRIS interface for seamless GNOME integration</summary>
```

Comment:
violates https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#not-technical

Reply:
you can have this in the description.